src/test/java/core/basesyntax/services/DataFileReaderTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.services;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataFileReaderTest{
+    private static final String FILE_PATH
+            = "src/test/resources/testFile.csv";
+    private static final String EMPTY_FILE_PATH
+            = "src/test/resources/emptyTestFile.csv";
+    private static final String NON_EXISTED_FILE_PATH
+            = "src/test/resources/111.txt";
+    private static DataFileReader fileReader;
+    private static List<String[]> expected;
+
+    @BeforeClass
+    public static void beforeClass(){
+        fileReader = new DataFileReader();
+    }
+
+    @Before
+    public void beforeEach(){
+        expected = new ArrayList<>();
+    }
+
+    @Test
+    public void readFromFile_ok(){
```

Comment:
don't use naming with underscore only in single method, that makes naming inconsistent
```suggestion
    public void readFromValidFile(){
```